src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationService.java

```diff
@@ -63,10 +63,10 @@ private void sendNotificationPerSubscription(NotificationWrapper notificationWra
             if (notificationWrapper.getNotificationType().isAllowOutOfHours() || !outOfHoursCalculator.isItOutOfHours()) {
                 sendNotificationService.sendEmailSmsLetterNotification(notificationWrapper, subscriptionWithType.getSubscription(), notification);
                 processOldSubscriptionNotifications(notificationWrapper, notification);
+                reminderService.createReminders(notificationWrapper);
             } else {
                 notificationHandler.scheduleNotification(notificationWrapper);
             }
-            reminderService.createReminders(notificationWrapper);
         }
```

Comment:
Not too sure I understand the specifics of reminders but doesn't reminderService.createReminders do more than scheduleNotification? If so would it not be better to change the else to also be

} else {
     if (isReninder) { //guess you can get this from the notificaitonWrapper somehow
                reminderService.createReminders(notificationWrapper);
     } else {
               notificationHandler.scheduleNotification(notificationWrapper);
    }
}